src/js/popup/popup.js

```diff
@@ -1736,6 +1739,32 @@
           },)
         }
 
+        // Show Holiday Promo 2023 News if the waffle flag is active and premium is available in user's country
+        if (isHolidayPromo2023Available && premiumAvailability) {
+          const getPeriodicalPremiumPlanId = getPeriodicalPremiumPlans.plan_country_lang_mapping[getPeriodicalPremiumPlans.country_code]["*"].yearly.id;
+          const fxaOrigin = (await browser.storage.local.get("fxaOrigin")).fxaOrigin.FXA_ORIGIN;
+          const holidayPromo2023Url =  `${fxaOrigin}/subscriptions/products/?plan=${getPeriodicalPremiumPlanId}&coupon=HOLIDAY20`;
```

Comment:
(Blocking)

I don't think this will direct to the correct link in prod.

The correct link is I got from prod is https://accounts.firefox.com/subscriptions/products/prod_KGizMiBqUJdYoY?plan=price_1LXUdlJNcmPzuWtRKTYg7mpZ ,

But when I use prod config by running npm run config:prod, the link from the add-on brings me to https://accounts.firefox.com/subscriptions/products/?plan=price_1LXUdlJNcmPzuWtRKTYg7mpZ and I get an error saying page not found.

Reply:
Added product ID - try again